controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -284,11 +289,13 @@ function execute($request) {
 
 			if ($issue) {
 
+				// set publication date
+				$submission->setDatePublished($this->getData('datePublished') ? $this->getData('datePublished') : Core::getCurrentDate());
```

Comment:
Hmmm... Would that date then be the (first) date_published for that submission? If an editor assigns a submission to a future issue, then unassigns it and assigns it later to the correct (and lets say published) issue, both dates will be in the DB (which I do not think is necessary?) and would then the first date be the date of article first publication (which would be wrong, I think, no?)?